src/openrpc/secure_storage.json

```diff
@@ -256,6 +256,168 @@
           }
         }
       ]
+    },
+    {
+      "name": "setForApp",
+      "summary": "Set or update a secure data value from distributor for an appId",
```

Comment:
It's not necessary going to be called by a distributor. Maybe change this to "for a specific app."